orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/strategies/RedBlackStrategy.groovy

```diff
@@ -80,6 +84,18 @@ class RedBlackStrategy implements Strategy, ApplicationContextAware {
       )
     }
 
+    if(stageData?.waitTime) {
+      def waitContext = [waitTime: stageData?.waitTime]
```

Comment:
`waitTime` seems like a possibly poorly named variable (as opposed to a name indicating it's a delayBeforeDisable or something?), I assume that's what deck is putting in there in rolling push and this is consistent with that?

Reply:
it's consistent with the naming in the wait stage